computedfields/resolver.py

```diff
@@ -849,7 +872,12 @@ def update_computedfields(self, instance, update_fields=None):
         changed based on the input fields, thus should extend `update_fields`
         on a save call.
         """
-        model = type(instance)
+        if not model:
+            model = type(instance)
+        parent_fields = self.update_computedfields_on_parents(instance, model, update_fields=update_fields)
+        if parent_fields and update_fields:
+            update_fields.update(parent_fields)
+
```

Comment:
Basically same concerns as above -  I am not sure yet, if those fields are good to be handled in `update_computedfields`, which is meant to deal with model local fields only (the whole MRO calculation relies on that).

Or if we need to defer their updates to foreign model handling with `update_dependent`.

Reply:
So we would walk the multi-table inheritance hierarchy in `update_dependent` rather than here ?

Reply:
> So we would walk the multi-table inheritance hierarchy in `update_dependent` rather than here ?

Not sure yet if we have to or if we can get this resolving correctly from the graph perspective. It def. would be nicer, if they can be handled locally, as the ORM pulls their values by hidden JOINs in the background on instance level. We have to replay that abstraction to some degree in the graph, not sure yet how cumbersome that is.